server/config/database.yml

```diff
@@ -19,9 +19,11 @@ default: &default
   encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
-  username: <%= ENV["DB_USER"] %>
-  password: <%= ENV["DB_PASSWORD"] %>
+  username: postgres
+  password: toor
```

Comment:
those changes should not be committed as your credentials looks different from setup of other developers, move those values into`.env.local` file
cc @kapatsila-mariia